### DISCUSSION TITLE
unhashable type: 'numpy.ndarray'

### DISCUSSION THREAD

**Discussion by loveinSH** [pybamm-team/PyBaMM#3463] [Q&A]
Hello,I am trying to change the equation of porosity by np.reshape and other numpy operations. And I want to use variables.update() to update the value. But is raises this problem:unhashable type: 'numpy.ndarray'. How can I fix it and update the value into variables?

**Comment by rtimms** [comment]
Can you please share a minimal working example to help show what is going wrong?
